Remotion/Development/UnitTesting.IsolatedCodeRunner.UnitTests/Development.UnitTesting.IsolatedCodeRunner.UnitTests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectType>Test</ProjectType>
+    <TargetFrameworks></TargetFrameworks>
+    <TargetFramework>net6.0-windows</TargetFramework>
```

Comment:
I think we should consider adding a flag for this. right now, it's only in WebApplication.props and now here. If we get more projects that are .net only, having a single declaration in remotion might be helpful. 

Also, the target frameworks override is declared before Shared.build.props, which means that they will get overridden again. This also gets solved by introducing a flag. `UseOnlyDotNetAsTargetFramework` or something of the sort. Not happy yet with the name.